src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -85,6 +87,9 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
         .toMat(Sink.queue[OfferMatchStatistics.OfferMatchUpdate])(Keep.both)
         .run
 
+    Mockito.when(instanceTracker.forceExpunge(any)).thenReturn(Future.successful(Done))
```

Comment:
we prefer the:
```suggestion
    instanceTracker.forceExpunge(any) returns Future.successful(Done)
```
notation.